src/middleware/filterResourcejsResponse.js

```diff
@@ -28,5 +28,10 @@ module.exports = (router) => (settings) => (req, res, next) => {
 
   res.resource.item = multi ? list : list[0];
 
+  /* eslint-disable callback-return */
+  if (callback) {
```

Comment:
we also need to check if callback's type is function to prevent callback is not a function error